repos/system_upgrade/common/files/upgrade_paths.json

```diff
@@ -1,16 +1,14 @@
 {
     "default": {
-        "7.6": ["8.4", "8.6"],
         "7.9": ["8.6", "8.8"],
         "8.6": ["9.0"],
-        "8.7": ["9.0"],
         "8.8": ["9.2"],
         "7": ["8.6", "8.8"],
         "8": ["9.2"]
     },
     "saphana": {
-        "7.9": ["8.2", "8.6"],
-        "7": ["8.2", "8.6"],
```

Comment:
@pirat89 keep 8.2 please. 8.2 life cycle is till mid 2024. We can drop it in November, ok?